packages/react-native-bpk-component-card/src/__snapshots__/withDivider-test.android.js.snap

```diff
@@ -1,404 +1,137 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Android withDivider should render correctly 1`] = `
-<BpkCard
```

Comment:
This looks like it is since we added the withTheme wrapper around the WithDivider component so it now exports the divider rather than it outputting a `BpkCard`